Removed the unnecessary create namespace command under section for setting up GCP pubsub source

rprakashg opened the pull request:
Removed kubectl command to create namespace "knative-sources" from section that covers setting up of GCP pubsub source as installation of eventing core sources already creates the namespace

Fixes #(issue-number)
#567 

## Proposed Changes
Removed the create namespace kubectl command that is not required from the README.md

RichieEscarez commented:
@rprakashg please take a look at and resolve the CLA issue. 

FYI, Ive run into this when sending a commit from computer that had an old email address in the config. Maybe this can help: https://help.github.com/articles/setting-your-commit-email-address-in-git/

rprakashg commented:
@RichieEscarez I fixed the email address issue through git but didn't think it would do anything with existing commits. I also went to the CLA site and added additional email. If I did another commit would it be enough for CLA check to clear?

evankanderson commented:
Thanks! It looks like if you install the eventing release, it will create
the namespace even if you don't install the regular (non-pubsub)
eventing-sources.

I'd originally written the tutorial for the case where you might install
just eventing-sources without the eventing release, which *does* need the
namespace and secret to be created. Too many install options...

On Tue, Nov 27, 2018 at 3:03 PM Knative Prow Robot <notifications@github.com>
wrote:

> Merged #568 <https://github.com/knative/docs/pull/568> into master.
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/knative/docs/pull/568#event-1991015113>, or mute the
> thread
> <https://github.com/notifications/unsubscribe-auth/AHlyN_Q5Za8faY0Y3c1hXSKPJqv_3cogks5uzcS_gaJpZM4YuxtO>
> .
>


-- 
Evan Anderson <argent@google.com>

RichieEscarez commented:
> Thanks! It looks like if you install the eventing release, it will create the namespace even if you don't install the regular (non-pubsub) eventing-sources. I'd originally written the tutorial for the case where you might install just eventing-sources without the eventing release, which *does* need the namespace and secret to be created. Too many install options...
> […](#)
> On Tue, Nov 27, 2018 at 3:03 PM Knative Prow Robot ***@***.***> wrote: Merged #568 <#568> into master. — You are receiving this because your review was requested. Reply to this email directly, view it on GitHub <[#568 (comment)](https://github.com/knative/docs/pull/568#event-1991015113)>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AHlyN_Q5Za8faY0Y3c1hXSKPJqv_3cogks5uzcS_gaJpZM4YuxtO> .
> -- Evan Anderson <argent@google.com>

I opened #603 to capture these details
